bulk-targeted-sponsorship/README.md

```diff
@@ -0,0 +1,381 @@
+# An approach for targeted bulk sponsorship of open source dependencies
+## A pseudo code explanation of how to programmatically identify a specific sample of an enterprise's dependencies for sponsorship
+
+This page gives a high level, pseudo code description of how Microsoft selected some of the projects in the June, 2024 FOSS Fund round #30. As the actual approach and code used is very specific to internal Microsoft systems, what is described here is an outline designed for others who might want to leverage a similar approach. 
+
+Please also read these for context: 
+- Microsoft Open Source blog post describing [5 things we learned from sponsoring a sampling of our open source dependencies](https://cloudblogs.microsoft.com/opensource/)
+- FOSSFUND repository [README](https://github.com/microsoft/foss-fund)
+- [Ecosystems page](https://opensource.microsoft.com/ecosystem/) on opensource.microsoft.com that describes FOSSFUND and other ways Microsoft supports the open source community including 
+Foundations, Azure Credits, etc.
+
+## Targeted bulk sponsorship
+### _An alternative to voting-based sponsorship or universal-thin-spread sponsorship_
+
+Traditionally, Microsoft's [FOSSFUND](https://github.com/microsoft/foss-fund) program has used
+a method of selecting open source for funding driven by employee recommendations and employee 
+voting. Employees who make upstream contributions to open source projects earn the right to
+vote on what projects get funded.
+This approach biases sponsorship to well known open source projects. 
+Another style of sponsorship is a univeral thin spread approach.
+All dependencies that have signed up for a particular funding
+platform are sponsored at the same level no matter their differences.
+
+As part of experimenting through different approaches, Microsoft trialed a data-driven
+approach in June of 2024 that leveraged the availability of good metadata on 
```

Comment:
how are we quantifying `good` here? either we need to define it or drop the qualifier.

Reply:
Took out 'good' as it works without.